site/app/controllers/GlobalController.php

```diff
@@ -343,20 +343,75 @@ public function header() {
         }
 
         $now = getDate(date_timestamp_get($this->core->getDateTimeNow()));
-        $month = $now['mon'];
-        $day = $now['mday'];
+        $duck_img = $this->getDuckImage($now);
```

Comment:
Can we just pass `$this->core->getDateTimeNow()` to `$this->getDuckImage()` and then use `$day = (int) $now->format('j')` and `$month = (int) $now->format('n`)` inside of the function.

Reply:
done